chainer/functions/math/sparse_matmul.py

```diff
@@ -419,13 +454,13 @@ def sparse_matmul(a, b, transa=False, transb=False):
     """
```

Comment:
Do you think we should document that 'C' order sparse matrices are faster on GPUs (for instance using `.. note::`)?